### DISCUSSION TITLE
How to re-enable GPS if I open Google Maps after I have started an OpenTracks track.

### DISCUSSION THREAD

**Discussion by mbrehon** [OpenTracksApp/OpenTracks#1524] [General]
Samsung s10e, Android v12, OpenTracks v 4.3.1-PlayStore.  Question is in title.  This may be an Android Q, but if there's a simple answer I won't need to register for a whole other forum site.

thanks,
-M

**Comment by dennisguse** [comment]
What actually happened?
I am not sure I understood it.
